### DISCUSSION TITLE
Automatically clearing a separator after removing characters after it

### DISCUSSION THREAD

**Discussion by bobleap** [uNmAnNeR/imaskjs#529] [Q&A] [answered]
Thanks for an amazing tool.

Question: is it possible to clear a separator after the characters after it have been removed?

To reproduce:
* Set up a mask using 00000[-0000]
* Type 6 digits and then enter backspace to remove the last
* Desired result: after 6th digit is removed, the hyphen is also removed
* Actual result: the hyphen remains

I also tried using blocks to see if the behavior was any different:

```
let patternMask = IMask(element, {
  mask: '00000[plus4]',
  blocks: {
    plus4: {
      mask: "-0000"
    }
  }
});
```
 
but no difference. Any ideas?

**Comment by uNmAnNeR** [comment, marked as answer]
there is an issue exists on that topic. In short: needs to be done.
as workaround you can use dynamic mask with 2 options: [ '00000', '00000-[0000]' ]

**Comment by bobleap** [comment]
Thanks! Tried "Mark as answer", but it appears to be broken.
